ElectronClient/InteropServiceHelper.js

```diff
@@ -96,6 +96,11 @@ class InteropServiceHelper {
 	}
 
 	static async defaultFilename(noteIds, fileExtension) {
+		// Use of == here is because noteIds is potentiall undefined
+		if (noteIds == null) {
+			return '';
+		}
```

Comment:
Not sure why you want to make the distinction between null and undefined? And also `null == undefined` anyway so I think you could simply have `!noteIds` here.

Reply:
I used == because I didn't want to make the distinction. But `!noteIds` will be just as good, I'll change it. Should I also change the `=== null` checks later in this function for conformity?

Reply:
Ideally yes, as usually we just check for truthy/falsy in this code base expect in rare cases. But for now just this fix is good enough. Thanks for the quick update!